lib/drip/client.rb

```diff
@@ -116,7 +116,9 @@ def make_request(verb_klass, uri, options, step = 0)
 
     def build_response(&block)
       response = yield
-      Drip::Response.new(response.code.to_i, response.body)
+      Drip::Response.new(response.code.to_i, response.body ? JSON.parse(response.body) : nil)
```

Comment:
Could `response.body` be an empty string? If so, could use `present?`. Or could `response.body` every be `false`?

Reply:
Empty string triggers the `JSON::ParserError` and returns `nil`, which is correct.

It can't be false, and if it were, the result should be `nil`.

`#present?` comes from ActiveSupport, which we don't use here.

I've improved the empty string case to not require an exception, and added a test (which passed before the change).